mmv1/products/redis/api.yaml

```diff
@@ -190,3 +190,43 @@ objects:
           - :STANDARD_HA
         default_value: :BASIC
         input: true
+      - !ruby/object:Api::Type::Enum
+        name: transitEncryptionMode
+        min_version: beta
+        input: true
+        description: |
+          The TLS mode of the Redis instance, If not provided, TLS is disabled for the instance.
+
+          - SERVER_AUTHENTICATION: Client to Server traffic encryption enabled with server authentcation
+        values:
+          - :SERVER_AUTHENTICATION
+          - :DISABLED
+        default_value: :DISABLED
+      - !ruby/object:Api::Type::Array
+        name: 'serverCaCerts'
```

Comment:
@slevenick any idea what this error is about? https://pantheon.corp.google.com/cloud-build/builds/eec8061e-2d56-4995-b3ae-be46aab37a82;step=15?project=graphite-docker-images

Reply:
done